lib/middleware/errors.js

```diff
@@ -24,14 +25,18 @@ function _applicationError(err, req, res, next) {
 function _authenticationError(err, req, res, next) {
   logger.log('warn', {'Unauthorized': 'Invalid user credentials'});
   logger.log('warn', {'Error stack': err.stack});
-  res.redirect(uriConstructor.redirectUrl(req.originalUrl));
+
+  return fetch(config.LOGOUT_URL, { headers: req.headers })
```

Comment:
Looking at that again I'm not entirly sure I understand why the res.redirect needs to be wrapped in a fetch. I think just redirecting to the LOGOUT_URL would solve the issue. I'm reading that as make a request ot LOGOUT_URL then redirect to LOGIN_URL I'm fairly sure LOGOUT_URL takes the same location parameter as LOGIN_URL